store/views.py

```diff
@@ -77,6 +145,64 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    
+    bookcopy_id = request.POST.get('bid')
+    bookcopy = get_object_or_404(BookCopy,id=bookcopy_id)
+    bookcopy.borrower = None
+    bookcopy.borrow_date = None
+    bookcopy.status = True
+    bookcopy.save()
+    print(request.POST)
+    msg = 'success'
+
+    response_data={
+        'message': msg
+    }
+
+    return JsonResponse(response_data)
```

Comment:
There must be a validation in the backend when a user is returning the book, to make sure that he has only borrowed the book. Otherwise, a simple POST request will make the BookCopy to be returned, and would set its status as True.